LICENSE

```diff
@@ -1,16 +1,16 @@
-Limerick Permissive License (LPL v2)
-
-(c) 2022 Alexander E Genaud
-
-Permission is granted hereby,
-to copy, share, use, modify,
-   for purposes any,
-   for free or for money,
-provided these rhymes catch eye.
-
-This work "as is" I provide,
-no warranty express or implied,
-   for, no purpose fit,
-   'tis unmerchantable shit.
-Liability for damages denied.
-
+ Limerick Permissive License (LPL v2)
```

Comment:
Was "Poetic License" which is ungoogleable. "Poetry" sounds refined and hauty, which is exactly not what I intend. After all it IS a "limerick".

I'm inspired by "public licenses" such as GPL and MPL, yet the LPL is far more similar to BSD and MIT, nearly identical to ISC, and not even the verbose Apache mentions "public license" anywhere. Perhaps no more nor less than "Limerick", "Permissive", and "License" describe what this word salad actually IS.